xAPI-Communication.md

```diff
@@ -1209,26 +1209,26 @@ a single defined document identified by "profileId". Otherwise, GET will return
 
 Example endpoint: http://example.com/xAPI/agents/profile
 
-Stores, changes, fetches, or deletes the specified profile document in the context of the specified Agent.  
+Stores, changes, fetches, or deletes the specified Profile document in the context of the specified Agent.  
```

Comment:
Absolutely agree, I was just normalizing the capitalization for the
intended use we've hinted at.  If they end up changing again, totally fine
with it.  It was just a hodgepodge in the current state.

On Fri, Aug 12, 2016 at 7:07 AM, Andrew Downes <notifications@github.com>
wrote:

> In xAPI-Communication.md
> <https://github.com/adlnet/xAPI-Spec/pull/975#discussion_r74574586>:
>
> > @@ -1209,26 +1209,26 @@ a single defined document identified by "profileId". Otherwise, GET will return
> >
> >  Example endpoint: http://example.com/xAPI/agents/profile
> >
> > -Stores, changes, fetches, or deletes the specified profile document in the context of the specified Agent.
> > +Stores, changes, fetches, or deletes the specified Profile document in the context of the specified Agent.
>
> I don't think capitalizing these goes far enough to solve the current open
> issue around this, but maybe that can can solved in a separate PR.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/adlnet/xAPI-Spec/pull/975/files/d7f9bc861173c32604a3ee834160341915b38624#r74574586>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABQ366FsZADpDdi7U9OoqlrZw3dc_1Yrks5qfFP1gaJpZM4JiiGq>
> .
>



-- 
Andy Johnson
ADL Technical Team
608-318-0049